src/pages/account/logins/index.tsx

```diff
@@ -208,16 +222,33 @@ export const Value = styled.p`
   margin-right: 0.5rem;
 `;
 
-type ButtonProps = {
-  color: string;
-};
+const Button = styled.button`
+  padding: 0.5rem;
+  margin-right: 1rem;
+  border-radius: ${({ theme }) => theme.radii[0]};
+  background-color: transparent;
+  color: ${({ theme }) => theme.colors.green};
+  transition: ${({ theme }) => theme.transitions.quick};
+  font-weight: ${({ theme }) => theme.fontWeights.medium};
+  &:hover {
+    cursor: pointer;
+    ${(props) => `background-color: ${props.color}`};
```

Comment:
You forgot the ones for `background-color`.

Reply:
Oops, fixed e6d1841